src/Components/Post/PostId.tsx

```diff
@@ -56,20 +62,24 @@ const PostId = () => {
   }, [id])
 
   useEffect(() => {
-    axios(`https://jsonplaceholder.typicode.com/comments`)
+    axios('https://jsonplaceholder.typicode.com/comments')
       .then((response) => {
-        setCommentData(response.data)
+        setCommentData(
+          response.data?.filter(
+            (comment: { postId: any }) => comment.postId === id,
+          ),
+        )
+        console.log(commentData.length)
       })
 
       .catch((error) => {
         console.error('Error in loading data', error)
         setError(error)
       })
-  }, [postData.id])
+  }, [])
 
-  const filteredComments = commentData.filter(
-    (comment: { [x: string]: unknown }) => comment['postId'] === id,
-  )
+  const count = commentData.length
+  console.log(commentData)
```

Comment:
can we take rid of these `console.log`? Generally speaking, you should version only production code.